crates/rolldown/tests/esbuild/dce/dce_of_using_declarations/artifacts.snap

```diff
@@ -0,0 +1,29 @@
+---
+source: crates/rolldown_testing/src/case/case.rs
+expression: content
+input_file: crates/rolldown/tests/esbuild/dce/dce_of_using_declarations
+---
+# Assets
+
+## entry_js.mjs
+
+```js
+
+//#region entry.js
+using null_keep = null;
+await using await_null_keep = null;
+using throw_keep = {};
+using dispose_keep = {[Symbol.dispose]() {
+	console.log('side effect');
+}};
+await using await_asyncDispose_keep = {[Symbol.asyncDispose]() {
+	console.log('side effect');
+}};
+using undef_keep = undefined;
+await using await_undef_keep = undefined;
+const Symbol_dispose_remove = Symbol.dispose;
+const Symbol_asyncDispose_remove = Symbol.asyncDispose;
```

Comment:
esbuild and rollup consider these two lines are side effects free, but webpack did not. 
Since `Symbol` could be overridden, we better consider these two lines have side effects.